src/Component/Movies.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Component } from "react";
+import '../App.css'
+
+
+class Movies extends Component{
+    constructor() {
+        super()
+        this.state = {
+            movies: []
+        }
+    }
+fetchFilms = () => {
+    fetch("https://ghibliapi.herokuapp.com/films")
+        .then((res) => res.json())
+        .then((data)=>{
+            this.setState({
+                movies: data,
+            })
+        });
+};
+    render () {
+        let showMovieArr = this.state.movies.map((movies)=>{
+            return (
+                <div className="movies-card">
+                    <ul>
+                       <li>Name: {movies.title}</li>
+                       <li>Release Date: {movies.releasedate}</li>
+                       <li>Description: {movies.description} </li>
+                    </ul>
+                </div>
+            )
+        })
+        return (
+          <div className='movies' >
+              <h1>Select a Movie</h1>
+              <option>
+
+              </option>
```

Comment:
```suggestion
              <???>

              </???>
```
You were on the right track here, but it's not exactly right. How do you make a dropdown menu using React/JavaScript? Try looking it up!